pkg/apis/planetscale/v2/vitesskeyspace_types.go

```diff
@@ -68,6 +68,13 @@ type VitessKeyspaceSpec struct {
 
 	// BackupEngine specifies the Vitess backup engine to use, either "builtin" or "xtrabackup".
 	BackupEngine VitessBackupEngine `json:"backupEngine,omitempty"`
+
+	// ExtraVitessFlags can optionally be used to override default flags set by the
```

Comment:
As in VitessCell, we can just say:

```go
// ExtraVitessFlags is inherited from the parent's VitessClusterSpec.
```

Reply:
Done https://github.com/planetscale/vitess-operator/pull/14/commits/226927d4cabfc3a79db1fcea1f2079d61b4ec302.